src/test/java/core/basesyntax/servise/ReportBuilderImplTest.java

```diff
@@ -0,0 +1,96 @@
+package core.basesyntax.servise;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.servise.exception.IncorrectOperationException;
+import core.basesyntax.servise.exception.InvalidInputDataException;
+import core.basesyntax.servise.inrterfase.ReportBuilder;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportBuilderImplTest {
+    private static final Fruit BANANA = new Fruit("banana");
+    private static final Fruit APPLE = new Fruit("apple");
+    private static List<FruitRecordDto> listFruitDto;
+    private ReportBuilder splitter;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        listFruitDto = new ArrayList<>();
+        listFruitDto.add(new FruitRecordDto(Operation.BALANCE, BANANA,20));
+        listFruitDto.add(new FruitRecordDto(Operation.BALANCE, APPLE, 100));
+        listFruitDto.add(new FruitRecordDto(Operation.SUPPLY, BANANA, 100));
+        listFruitDto.add(new FruitRecordDto(Operation.PURCHASE, BANANA, 13));
+        listFruitDto.add(new FruitRecordDto(Operation.RETURN, APPLE, 10));
+        listFruitDto.add(new FruitRecordDto(Operation.PURCHASE, APPLE, 20));
+        listFruitDto.add(new FruitRecordDto(Operation.PURCHASE, BANANA, 5));
+        listFruitDto.add(new FruitRecordDto(Operation.SUPPLY, BANANA, 50));
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        splitter = new ReportBuilderImpl();
```

Comment:
if ReportBuilderImpl() doesn't have any state we can init it in BeforeClass, no need to create separate instance per each test

Reply:
done